app/queue_publisher.py

```diff
@@ -56,7 +56,7 @@ def _publish(self, message, content_type=None, headers=None):
             self._logger.error("Unable to publish message", exception=repr(e))
             return False
 
-    def publish_message(self, message, content_type=None, headers=None):
+    def publish_message(self, message, content_type=None, headers=None, **kwargs):
```

Comment:
It means you can pass a secret into the non-encrypting QueuePublisher and it won't mind. Not a realistic scenario, so I'll take it out.